src/Groups/directproducts.jl

```diff
@@ -372,10 +382,12 @@ Group homomorphism
   to aut( <pc group of size 8 with 3 generators> )
 
 julia> G = semidirect_product(Q,f,C)
-SemidirectProduct( <pc group of size 8 with 3 generators> , <pc group of size 2 with 1 generator> )
+Semidirect product of
+  pc group of order 8
+  pc group of order 2
```

Comment:
just as an idea: instead of printing a list here, add something like `acting on` in-between to make it clear which of the two group acts on which

Reply:
the second group acts on the first, so it would have to be something awkward like `being acted upon by` ... or maybe "normalized by" ... ho-hum

Reply:
Ignoring the exact phrase, I also wonder how to format it:
1.
```suggestion
Semidirect product of
  pc group of order 8
normalized by
  pc group of order 2
```
2.
```suggestion
Semidirect product of
    pc group of order 8
  normalized by
    pc group of order 2
```
3.
```suggestion
Semidirect product of
  pc group of order 8
  normalized by
  pc group of order 2
```
4.
```suggestion
Semidirect product of
  pc group of order 8 normalized by
  pc group of order 2
```
5.
```suggestion
Semidirect product of
  pc group of order 8
  normalized by pc group of order 2
```
6. ... ???

Reply:
Any thoughts on this @ThomasBreuer @lgoettgens ?

Reply:
I don't have a strong opinion between any of your suggestions